include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_scan.h

```diff
@@ -64,20 +67,23 @@ struct __scan_status_flag
         for (int tile = static_cast<int>(tile_id) + offset; tile >= 0; tile -= 32)
```

Comment:
We could be `good boys` and define `constexpr int subgroup_size = 32` somewhere for now? We use it as a magic number in a few places I think.

Reply:
I've added a var in the namespace `igpu`